packages/react-self-serializers/CHANGELOG.md

```diff
@@ -0,0 +1,51 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
```

Comment:
Replace this line with something like. Same with the other package


## ❗️ Deprecated on 2019-09-10 on version 1.3.15 ❗️